requirements/requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 accelerate
 clang-format==16.0.2
+comet_ml>=3.41.0
```

Comment:
@mrwyattii - thoughts on moving things like comet_ml, wandb, etc for monitoring to a new requirements file?

Reply:
Discussed offline, this is fine.